### DISCUSSION TITLE
Max zoom

### DISCUSSION THREAD

**Discussion by tallano** [Leaflet/Leaflet#8122] [Q&A] [answered]
Hi,
How do you zoom beyond the max zoom offered by the data source you're working on? From what I've read, it looks like it is possible, but I couldn't find how to do it.

**Comment by IvanSanchez** [comment]
I'll answer with a question: What's the default value of the `maxZoom` option of a `L.TileLayer`?

**Comment by tallano** [comment]
I am sorry, I'm pretty much of a beginner with this, and though I studied the documentation, I don't know the answer to your question. I tried logging the maxZoom value of the tile I'm loading, and it is 21. But when I zoom, the layer vanishes at zoom 20.

**Comment by Falke-Design** [comment, marked as answer]
You need to use [maxNativeZoom](https://leafletjs.com/SlavaUkraini/reference.html#gridlayer-maxnativezoom). It makes that if the zoom level goes over the `maxNativeZoom` value, it scales up the image from the maxNativeZoom level instead of requesting the tile from the server, 

```
L.tileLayer(url, {maxZoom: 21, maxNativeZoom: 20});
```

**Comment by tallano** [comment]
It works, thank you so much!
